zeropilot3.5/include/system_manager/system_manager.hpp

```diff
@@ -6,17 +6,18 @@
 
 class SystemManager {
     public:
-    SystemManager(SBusIface *rc_driver, QueueIface<RCMotorControlMessage_t> *queue_driver);
+    SystemManager(ISBUSReceiver *rc_driver, IMessageQueue<RCMotorControlMessage_t> *queue_driver, int16_t invalid_threshold=50);
     ~SystemManager();
```

Comment:
remove `~SystemManager();` and leave the destructor implicit if you are not doing anything special with it